File: app/src/main/java/za/co/xisystems/itis_rrm/ui/mainview/corrections/CorrectionsViewModel.java
package za.co.xisystems.itis_rrm.ui.mainview.corrections;


import androidx.lifecycle.LiveData;
import androidx.lifecycle.MutableLiveData;
import androidx.lifecycle.ViewModel;

public class CorrectionsViewModel extends ViewModel {

    private MutableLiveData<String> mText;

    public CorrectionsViewModel() {
        mText = new MutableLiveData<>();
        mText.setValue("This is Estimates to be Corrected fragment");
    }

    public LiveData<String> getText() {
        return mText;
    }
}

File: app/src/main/java/za/co/xisystems/itis_rrm/ui/mainview/corrections/CorrectionsViewModel.kt
package za.co.xisystems.itis_rrm.ui.mainview.corrections

import androidx.lifecycle.ViewModel
import za.co.xisystems.itis_rrm.data.repositories.OfflineDataRepository
import za.co.xisystems.itis_rrm.utils.lazyDeferred

class CorrectionsViewModel (
    private val offlineDataRepository: OfflineDataRepository
) : ViewModel() {

    val offlinedata by lazyDeferred {
        offlineDataRepository.getSectionItems()
        offlineDataRepository.getContracts()
    }
}